examples/hypercompute_clusters/a3u-gke-gcs/kueue-configuration.yaml.tftpl

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ResourceFlavor
+metadata:
+  name: "default-flavor"
+---
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+metadata:
+  name: "cluster-queue"
+spec:
+  namespaceSelector: {}
+  resourceGroups:
+  - coveredResources: ["nvidia.com/gpu"]
+    flavors:
+    - name: "default-flavor"
+      resources:
+      - name: "nvidia.com/gpu"
+        nominalQuota: ${num_gpus}
+
```

Comment:
In line with using Kueue v0.10.0 as the recommended solution, please modify this configuration file to be similar to the the [tas-queues.yaml](https://github.com/ighosh98/cluster-toolkit/blob/8c26d4a1aac024b0ce848e5e1ab7b962b8e1812b/tools/cloud-build/daily-tests/blueprints/kueue-config-files/tas-queues.yaml) file

Also, have we tried using this configuration to provision a cluster and schedule a workload?